archive.sh

```diff
@@ -8,6 +8,18 @@ touch ${TRAVIS_COMMIT}
 # Selecting only a file in travis doesn't seem to work, using a dir instead
 mkdir -p to_upload
 
+# Move the pt_BR files to pt so Django can find them
+# Foundation site
+rm -rf ./foundation/translations/locale/pt/
+mv ./foundation/translations/locale/pt_BR/ ./foundation/translations/locale/pt/
+# Buyers' guide
+rm -rf ./foundation/translations/networkapi/buyersguide/locale/pt/
+mv ./foundation/translations/networkapi/buyersguide/locale/pt_BR/ ./foundation/translations/networkapi/buyersguide/locale/pt/
+# Mozfest
```

Comment:
I ended up splitting the PNI strings in two locations, so let’s move this one too

```suggestion
rm -rf ./foundation/translations/networkapi/buyersguide/templates/about/locale/pt/
mv ./foundation/translations/networkapi/buyersguide/templates/about/locale/pt_BR/ ./foundation/translations/networkapi/buyersguide/templates/about/locale/pt/
# Mozfest
```